extensions/web/09_file.js

```diff
@@ -189,79 +190,88 @@
      * @param {string} [contentType]
      * @returns {Blob}
      */
-    slice(start, end, contentType) {
+    slice(start = 0, end = this.size, contentType = '') {
```

Comment:
```suggestion
    slice(start = undefined, end = undefined, contentType = undefined) {
```

The webidl for slice is this:
```
  Blob slice(optional [Clamp] long long start,
            optional [Clamp] long long end,
            optional DOMString contentType);
```

The "optional" prefix for a field means it should be `undefined` if nothing is specified.

You will also need to re-add the `if (start !== undefined) { ` below.

Reply:
Tried dealing with undefined before, resulted in getting `NaN` values
using default `arguments = undefined` don't make much sense...

Can revert and simply do `start || 0`